the-collab-lab/tcl-19-smart-shopping-list
# Add SEO and OGP metag tags

## Description

This PR adds OGP and SEO meta tags to `index.html`.


## Related Issue

Closes #36 


## Acceptance Criteria


There should be OGP and SEO meta tags in `index.html`.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria


- Pull this branch from gitHub using `git pull origin jm-cu-ogp-seo-meta-tags`
- There should be SEO and OGP meta tags in `index.html`
